event-details-boundary/Menu.test.tsx

```diff
@@ -188,6 +197,10 @@ describe("EventDetailsMenu tests", () => {
       })
     })
 
+    const ensureUserIsSignedIn = () => {
```

Comment:
nit - can we call it something like "useSignedInUser", "ensure" sounds like we're checking a condition